go/obscuronode/host/hostrunner/host_runner.go

```diff
@@ -53,7 +52,7 @@ func RunHost(config HostConfig) {
 func SetLogs(logPath string) {
 	logFile, err := os.Create(logPath)
 	if err != nil {
-		panic(err)
+		panic(fmt.Sprintf("could not create log file. Cause: %s", err))
```

Comment:
here

Reply:
We can't log anything here as the problem is we can't create a log file.

Reply:
It will log to the default console logger I think :)

Reply:
But the panic will go there too anyway